graylog2-web-interface/src/views/components/sidebar/highlighting/HighlightingRule.jsx

```diff
@@ -30,19 +68,20 @@ const onDelete = (e, rule) => {
 const HighlightingRule = ({ rule }: Props) => {
   const { field, value, color } = rule;
   return (
-    <div className={styles.highlightingRuleBlock}>
+    <HighlightingRuleGrid>
       <ColorPickerPopover id="formatting-rule-color"
                           placement="right"
                           color={color}
                           colors={DEFAULT_CUSTOM_HIGHLIGHT_RANGE.map((c) => [c])}
-                          triggerNode={<div className={styles.colorElement} style={{ backgroundColor: color }} />}
+                          triggerNode={<ColorPreview color={color} />}
                           onChange={(newColor, _, hidePopover) => updateColor(rule, newColor, hidePopover)} />
-      {' '}
-      for <strong>{field}</strong> = <i>&quot;{value}&quot;</i>.
-      <span role="presentation" title="Remove this Highlighting Rule" onClick={(e) => onDelete(e, rule)}>
+      <div>
+        for <strong>{field}</strong> = <i>&quot;{value}&quot;</i>.
+      </div>
```

Comment:
I had to use a `<div>` instead of a `<span>` here, otherwise the grid will not be displayed correctly with IE11.